lib/ansible/utils/module_docs_fragments/spacewalk.py

```diff
@@ -0,0 +1,33 @@
+# Copyright: (c) 2018, Davide Blasi (@davegararh)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+    # Parameters for Sapcewalk modules
+    DOCUMENTATION = '''
+options:
+    url:
+      description:
+      - The full URL to the Spacewalk API such as
+      - C(https://spacewalk.example.net/rpc/api)
+      - you can use eviroment variable C(SPACEWALK_URL)
+      type: str
+      required: true
+    login:
+      description:
+      - Spacewalk login.
+      - you can use eviroment variable C(SPACEWALK_LOGIN)
+      type: str
+      required: true
+    password:
```

Comment:
Does Spacewalk support any API keys or certificates instead of username/password combinations?

Reply:
I don't seen documentation about key o certificate for api. I try to ask in channel IRC or mailing list

Reply:
I confirm that auth is handled in user,password only, and no REST api